src/shared-components/container/style.ts

```diff
@@ -15,7 +15,7 @@ const clickableStyle = (theme: ThemeType) => `
 `;
 
 const messageStyle = (theme: ThemeType) => `
-  border-radius: 16px;
+  border-radius: ${theme.BORDER_RADIUS.medium};
```

Comment:
I am okay with this change but it is probably worth double-checking with Ben that this does not conflict with what is present throughout Design prototypes in Figma: https://www.chromatic.com/test?appId=5f6e46f1e7081000221523cf&id=5fd8d768e3a0560021add39c

Reply:
this was approved by Ben, in my opinion it looks even better with 8px